subiquity/models/kernel.py

```diff
@@ -40,11 +40,9 @@ def needed_kernel(self) -> Optional[str]:
         return self.metapkg_name
 
     def render(self):
+        result = {"kernel": {"remove_existing": True}}
         if self.curthooks_no_install:
```

Comment:
I don't think any core remains to set curthooks_no_install to True, so can this be tidied up?

Reply:
I feel tentative on if it's going to be useful or not, so I'm dropping it but isolating that change to a distinct commit.